app/views/items/show.html.erb

```diff
@@ -30,20 +30,49 @@
         <div class="card">
           <div class="card-body p-0">
             <table class="table">
-              <thead>
+            <tbody>
               <tr>
-                <th>Value Per Item</th>
-                <th>Distribution Quantity</th>
-                <th>Package Size</th>
+                <th>Base Item</th>
+                <td><%= @item.base_item.name ? @item.base_item.name : '' %></td>
+              </tr>
+              <tr>
+                <th>Name</th>
+                <td><%= @item.name ? @item.name : '' %></td>
+              </tr>
+              <tr>
+                <th>Category</th>
+                <td><%= @item&.item_category&.name %></td>
               </tr>
-              </thead>
-              <tbody>
               <tr>
+                <th>Value Per Item</th>
                 <td><%= @item.value_in_cents ? @item.value_in_cents : 0 %></td>
+              </tr>
+              <tr>
+                <th>Quantity Per Indivudual</th>
                 <td><%= @item.distribution_quantity ? @item.distribution_quantity : 0 %></td>
+              </tr>
+              <tr>
+                <th>On hand minimum quantity</th>
+                <td><%= @item.on_hand_minimum_quantity ? @item.on_hand_minimum_quantity : 0 %></td>
+              </tr>
+              <tr>
+                <th>On hand recommended quantity</th>
+                <td><%= @item.on_hand_recommended_quantity ? @item.on_hand_recommended_quantity : 0 %></td>
+              </tr>
+              <tr>
+                <th>Package Size</th>
                 <td><%= @item.package_size ? @item.package_size : 0 %></td>
               </tr>
-              </tbody>
+              <tr>
+                <th>Custom Units</th>
+                <% item_units = @item.request_units&.pluck("item_units.name") %>
+                <td><%=item_units ? item_units.join("; ") : ''%></td>
+              </tr>
+              <tr>
```

Comment:
Per the issue,  'The Custom request units bit must be behind the flipper flag "enable_packs" '.

Reply:
could you please explain this a little bit.

Reply:
`              <tr>
                <th>Custom Units</th>
                <% item_units = Flipper.enabled?(:enable_packs) ? @item.request_units&.pluck("item_units.name") : [] %>
                <td><%=item_units ? item_units.join("; ") : ''%></td>
              </tr>`

Reply:
The whole section,  including the title, has to be behind the flag --- we aren't releasing that feature until we've finished all the pieces that need to be present for the banks to use custom units.

Reply:
Hey @Naraveni -- Looks like I was unclear -- I meant just the custom request units section, not all the item information.   Thanks.